docs/src/administration/users.md

```diff
@@ -1,65 +1,60 @@
 ---
-title: "User administration"
+title: "Users"
 weight: 1
-sidebarTitle: "Users"
+sidebarTitle: "User administration"
 description: |
-  Every Platform.sh user has a role that controls their access and permission levels. Different roles allow different levels of access to your applications, environments and projects. You can manage how users interact with your project and environments at Platform.sh
+  Every Platform.sh user has a role that controls their access and permission levels.
 ---
 
 {{< description >}}
 
-Any user added to a project or an environment type on Platform.sh will need to [register for an account](https://auth.api.platform.sh/register) before they can contribute. If you need to delete your account at any time you can [transfer ownership](/administration/users.md#transfer-ownership) of your existing projects and then open a [support ticket](/development/troubleshoot.md#deleting-your-platformsh-account) to request your account deletion.  
+If you are an organization or a Project admin wanting to add a user to a project or an environment, the user has to create an account before they can contribute to the project.
 
 ## User roles
 
-Grant user permissions to the entire project:
-* **Project Viewer** - A project viewer can view all environments of the project.
-* **Project Administrator** - A project administrator can change settings, push code and execute actions on all environments of the project.
+| User role    | Description |
+| ------------ |-------------|
+|Project owner| There is only one and they can create multiple Project admins.  |
+|Project admin | Users who can configure project settings, manage other users,  administer environment permissions, push code, and execute actions on all project environments.|
 
-You can also define user access per environment type (Production, Staging, Development). Each permission level progressively increases access to the environments part of a given environment type:
+### Transfer a project ownership
```

Comment:
```suggestion
### Transfer project ownership
```
Also, why is this a section within "user roles"? It seems to be a combination of reference with how to that might be confusing. A clearer goal from the start might help.

Reply:
Changed.